client/src/pages/participant/Research.vue

```diff
@@ -17,8 +17,39 @@
                                 My Studies
                             </h2>
                         </div>
-                        List the studies here...
+                        <div class="flex mb-2">
+                          <table>
```

Comment:
Does using the `table` HTML element here allow for easy responsive design?

Reply:
No - it locks us into using that HTML element. 
Ideally, we would have `div`s that change their style depending on the device, to provide a really nice, user-friendly layout.

In this case, I checked this current implementation in Responsive Design Mode in Firefox, and it is usable. 
The reason I went for the table element is that I wasn't able to adapt to your custom CSS quickly, and I didn't want to import additional libraries like MaterialUI and create inconsistencies with your design.
Under normal circumstances, I'd probably ask someone on the frontend team to walk me through your styles and practices.